reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -7526,6 +7571,11 @@ public boolean test(O o) {
 	}
 
 	@SuppressWarnings("unchecked")
+	static <U, V> BiPredicate<U, V> equalPredicate() {
```

Comment:
this equals function change is orthogonal... shelve it and isolate it in another PR / commit on master